crates/api/src/post/lock.rs

```diff
@@ -26,9 +27,10 @@ pub async fn lock_post(
   let post_id = data.post_id;
   let orig_post = Post::read(&mut context.pool(), post_id).await?;
```

Comment:
Same here, could read the post_view rather than multiple reads.

Reply:
Done